docs/developer/checkout/problems.mdx

```diff
@@ -4,19 +4,6 @@ title: Problems
 
 ## Introduction
 
-:::info
-
-This feature was introduced in **Saleor 3.15**.
-
-:::
-
-:::caution
-
-This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
```

Comment:
Shouldn't we do this in core first? Right now we will have mismatch between docs, and `API-references` generated based on the core's schema.

In case of checkout.problems, not all potential error events are covered, this is why we keep this in preview 🤔  (at least this is my understanding).